pkgs/nbxplorer/result

```diff
@@ -0,0 +1 @@
+/nix/store/bxwbv6xfgcl173zjfixvmk1bjirgf0la-NBXplorer
```

Comment:
Why is this file included?

Reply:
Yeah that doesn't need to be there. I'm doing cleanup at the moment.